src/ItkVtkViewProxy.js

```diff
@@ -93,6 +93,14 @@ function ItkVtkViewProxy(publicAPI, model) {
     }
   }
 
+  function getAnnotationText(value) {
+    const labelValue = value[model.labelIndex]
```

Comment:
Should this function live here and the `annotationMap` and `labelIndex` provided to this thing's `model`?  Or should we just provide this proxy class with a method that can be called to retrieve the annotation text?

Reply:
This looks good to me :+1: